benchmarks/index.js

```diff
@@ -272,3 +122,141 @@ class SimpleRequest {
     throw err
   }
 }
+
+function makeParallelRequests (cb) {
+  return Promise.all(Array.from(Array(parallelRequests)).map(() => new Promise(cb)))
+}
+
+cronometro(
+  {
+    'http - no agent' () {
+      return makeParallelRequests(resolve => {
+        http.get(httpNoAgent, res => {
```

Comment:
```suggestion
        http.get(httpNoKeepAlive, res => {
```

Make an agent that has the `maxSockets` options but `keepAlive: false`